File: python/BRIDGE_WHOLE_PART/BRIDGE_WHOLE_PART/bridge/ProductoCine.py
from abc import ABC, abstractmethod

class ProductoCine(ABC):
    
    tipoProductoCine : int
    
    def __init__(self, tipoProductoCine) -> None:
        self.tipoProductoCine = tipoProductoCine
    
    @abstractmethod
    def crearProductoCine(self):
        pass

File: php/BRIDGE_WHOLE_PART/bridge/ProductoCine.php
<?php


abstract class ProductoCine {
    
    public $tipoProductoCine;
    
    public function __construct($tipoProductoCine) {
        $this->tipoProductoCine = $tipoProductoCine;
    }
    
    public function crearProductoCine() {
        
    }
    
}
